sdk/core/azure-core-amqp/src/main/java/com/azure/core/amqp/AmqpTransaction.java

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.core.amqp;
+
+import java.nio.ByteBuffer;
+
+/**
+ * Represents transaction.
+ */
+public class AmqpTransaction {
+
+    private final ByteBuffer transactionId;
```

Comment:
Does it have to be a bytebuffer? In your usages of it, I see you getting byte[]

Reply:
We have compile time check which does not allow `com.azure.core.amqp.AmqpTransaction.getTransactionId() may expose internal representation by returning AmqpTransaction.transactionId`  and I do not want to copy them in constructor and in getter.